_posts/en/newsletters/2022-02-23-newsletter.md

```diff
@@ -0,0 +1,139 @@
+  sponsorship][topic fee sponsorship] (previously described in
+  [Newsletter #116][news116 sponsorship]).
+
+    The idea received significant discussion on the mailing list.
+
+- **Updated LN gossip proposal:** Rusty Russell [posted][russell gossip]
+  to the Lightning-Dev mailing list a detailed proposal for a new set of
+  LN gossip messages similar to his 2019 proposal described in
+  [Newsletter #55][news55 gossip].  The new proposal uses
+  [BIP340][]-style [schnorr signatures][topic schnorr signatures] and
+  [x-only public keys][news72 xonly].  Also included are a number of
+  simplifications over the existing LN gossip protocol, which is used to
+  advertise the existence of public channels for routing.  The updated
+  protocol is designed to maximize efficiency, especially when combined
+  with an [erlay][topic erlay]-like [minisketch][topic minisketch]-based
+  efficient gossip protocol.
+
+- **CTV signet:** Jeremy Rubin [published][rubin ctv signet] parameters
+  and code for a [signet][topic signet] with
+  [OP_CHECKTEMPLATEVERIFY][topic op_checktemplateverify] activated.
+  This simplifies public experimentation with the proposed opcode and
+  makes it much easier to test compatibility between different software
+  using the code.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Will a post-subsidy block with no transactions include a coinbase transaction?]({{bse}}112193)
+  Pieter Wuille explains that since every block must have a coinbase transaction
+  and since every transaction must include at least one
+  input and one output, a post-subsidy block with no block reward (no fees and
+  no subsidy) will still require at least one zero-value output.
+
+- [How can the genesis block contain arbitrary data on it if the script is invalid?]({{bse}}112439)
+  Pieter Wuille lists the reasons why the genesis block's coinbase
+  "Chancellor..." text push is valid. First, the [genesis block][bitcoin se 13122] is valid by
+  definition. Second is that coinbase input scripts are not actually executed.
+  Third is that, pre-[BIP342][], the requirement of a single element on the stack
+  after execution is only a policy rule, not consensus. Finally, that policy
+  rule applies only when executing both input and output scripts and since the
+  genesis block's input does not have an output script, the policy does not
+  apply. Pieter also notes the reason for the genesis block's unspendability is
```

Comment:
```suggestion
  rule applies only to the final stack after an input script is executed together with the corresponding output script. Since there 
  are no corresponding output scripts for the inputs of coinbase transactions, the policy does not
  apply. Wuille also notes the reason for the genesis block's unspendability is
```

Reply:
(we refer to people with their last name above, so I changed it here as well)